kokoro/linux/64-bit/Dockerfile

```diff
@@ -66,6 +66,8 @@ RUN apt-get clean && apt-get update && apt-get install -y --force-yes \
   openjdk-7-jdk \
   # -- For java_oracle7 -- \
   oracle-java7-installer \
+  # -- For java_linkage_monitor \
```

Comment:
Have you tried to build this image locally? I remember there was some issue (obsolete dependency) blocking the build.

Reply:
No, let me try that.

Reply:
I prefer to create a new docker file and put it under kokoro/linux/dockerfile/test/java

Reply:
This Dockerfile cannot be built as of now. Created an issue https://github.com/protocolbuffers/protobuf/issues/6314 .